src/search/FindInFilesUI.js

```diff
@@ -92,7 +94,7 @@ define(function (require, exports, module) {
                         _findBar.enable(true);
                         _findBar.focusQuery();
                         if (zeroFilesToken === FindInFiles.ZERO_FILES_TO_SEARCH) {
-                            _findBar.showError(StringUtils.format(Strings.FIND_IN_FILES_ZERO_FILES, FindUtils.labelForScope(FindInFiles.searchModel.scope)), true);
+                            _findBar.showError(StringUtils.format(Strings.FIND_IN_FILES_ZERO_FILES, FindUtils.labelForScope(searchScope)), true);
```

Comment:
This is for the second part of the issue #9974 where Error message displayed **project** instead of the current folder name.